### DISCUSSION TITLE
blog/2024/02/10/cmd/mix/

### DISCUSSION THREAD

**Discussion by giscus** [MarsZone/marsblog#9] [Announcements]
# blog/2024/02/10/cmd/mix/

个人博客

http://marsco.online/blog/2024/02/10/cmd/mix/

**Comment by MarsZone** [comment]
补个链接，https://zhuanlan.zhihu.com/p/684961969

**Reply by MarsZone** [reply]
最新：ChatGPT-3.5/4.0ALL国内中文镜像网站
https://zhuanlan.zhihu.com/p/678199581
